examples/getting-started-movielens/03-Training-with-HugeCTR.ipynb

```diff
@@ -57,7 +57,7 @@
     "HugeCTR is a GPU-accelerated recommender framework designed to distribute training across multiple GPUs and nodes and estimate Click-Through Rates (CTRs).<br>\n",
```

Comment:
<p>10x speed up in training? with what dataset and model?DLRM? do you mind to give just one line more explanation or link to prove that statement? thanks.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/779/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
This information was there in the notebook, someone else is stating that they saw this improvement, I have not come across any supporting evidence.

Should we keep it as is? otherwise can remove the 10x? or the whole passage altogether?

Reply:
@bschifferer  do you know any supporting evidence/link of 10X speed up? thanks.